src/Microsoft.OData.Edm/EdmModelVisitor.cs

```diff
@@ -427,21 +455,45 @@ protected virtual void ProcessElement(IEdmElement element)
             this.VisitAnnotations(this.Model.DirectValueAnnotations(element));
         }
 
+        protected virtual Task ProcessElementAsync(IEdmElement element)
+        {
+            // TODO: DirectValueAnnotationsInMainSchema (not including those in referenced schemas)
+            this.VisitAnnotations(this.Model.DirectValueAnnotations(element));
+
+            return Task.FromResult(0);

```

Comment:
Why `Task.FromResult(0)` instead of `Task.CompletedTask`? Do we do this in the ODL 8 branch as well?